CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Add new threshold parameters to monitor config ([#3181](https://github.com/getsentry/sentry-java/pull/3181))
+- Configurable defaults for Cron - MonitorConfig ([#3195](https://github.com/getsentry/sentry-java/pull/3195))
```

Comment:
`m` should we also add the new config options added in https://github.com/getsentry/sentry-java/pull/3181/files ?